File: AbstractFactoryDesignPattern/NotificationService_Example/Message/SMSMessage.java
package AbstractFactoryDesignPattern.NotificationService_Example.Message;

public class SMSMessage implements Message {
	String toPhoneNum;
	String body;

	@Override
	public void setContent(String to, String body) {
		this.toPhoneNum = to;
		this.body = body.length() > 160 ? body.substring(0, 160) : body;
	}

	@Override
	public String formatMessage() {
		return "SMS to " + toPhoneNum + ": " + body;
	}
}

File: AbstractFactoryDesignPattern/NotificationService_Example_Python/Message/SMSMessage.py
from AbstractFactoryDesignPattern.NotificationService_Example_Python.Message.NotificationMessage import NotificationMessage

class SMSMessage(NotificationMessage):

	def __init__(self) -> None:
		self.toPhoneNum : str = ""
		self.body: str = ""

	def setContent(self, to : str, body : str) -> None:
		self.toPhoneNum = to
		self.body = body[:160]

	def formatMessage(self) -> str:
		return "SMS to " + self.toPhoneNum + ": " + self.body